### ISSUE TITLE
Support entries like "mod 'pcfens-filebeat', '0.10.3'"

### ISSUE THREAD

**Issue by gdubicki** [xorpaul/g10k#40]
Currently g10k fails to deploy Puppetfile with such entry with this error:

```
2017/03/15 21:43:12 Error: Missing :git url in /etc/puppet/environments/_myenv/Puppetfile for module pcfens-filebeat line: mod 'pcfens-filebeat', '0.10.3'
```

I suppose that it would accept it if it was `mod 'pcfens/filebeat', '0.10.3'` (slash instead of dash).

**Comment by xorpaul** [comment]
Does r10k really support 
```
mod 'pcfens-filebeat', '0.10.3' 
```
?

I've never seen something like that in a Puppetfile for a Forge module.
I'll have to check that out.

**Comment by maxadamo** [comment]
@xorpaul yes it's supported in r10k :) That's what you copy and paste from puppetforge: the name with the hyphen, and not slash

**Comment by gdubicki** [comment]
Like @maxadamo said, it comes from the forge itself. :/ Works in current released r10k, 2.5.1 IIRC.

**Comment by xorpaul** [comment]
Okay, try out v0.2.9b https://github.com/xorpaul/g10k/releases/tag/v0.2.9b

PR: #42

**Comment by gdubicki** [comment]
This seems to be solved, thank you @xorpaul ! But I encountered a new issue with 0.2.9b and it may be related - will report it in a moment.

**Comment by gdubicki** [comment]
I mean this: https://github.com/xorpaul/g10k/issues/46

**Comment by xorpaul** [comment]
closed with #42
